[Azure AD] [ELM] added attribute resource docs (#13931)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accesspackageresourceattributequestion.md

---
title: "accessPackageResourceAttributeQuestion resource type"
description: "A question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system and/or request approver"
author: "markwahl-msft"
localization_priority: Normal
ms.prod: "Governance"
doc_type: resourcePageType
---

# accessPackageResourceAttributeQuestion resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

A question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system and/or request approver. 

Inherits from [accessPackageAttributeSource](../resources/accesspackageattributesource.md).

## Properties
None.

## Relationships
None.

## JSON representation
The following is a JSON representation of the resource.
<!-- {
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessPackageAttributeQuestion"
}
-->
``` json
{
  "@odata.type": "#microsoft.graph.accessPackageAttributeQuestion",
  "question": {
    "@odata.type": "microsoft.graph.accessPackageQuestion"
  }
}
```

> ## Example 2: Get accessPackagePolicy

Reviewer: Should this go in the accesspackageassignmentpolicy doc instead?

Author: I removed this. It wasn't one of the examples in the latest note

---
title: "accessPackageResourceAttributeQuestion resource type"

> description: "A question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system and/or request approver"

Reviewer 2: ```suggestion
description: "Resource that defines the question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system or the request approver."
```

author: "markwahl-msft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessPackageResourceAttributeQuestion resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system and/or request approver. 

Reviewer 2: ```suggestion
Resource that defines the question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system or the request approver.
```

Inherits from [accessPackageResourceAttributeSource](../resources/accesspackageresourceattributesource.md).

## Properties

> None.

Reviewer 2: **question** property of type **accessPackageQuestion** is supported by this API.

Author: @NelsonBlakeN

---
title: "accessPackageResourceAttributeQuestion resource type"
description: "Resource that defines the question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system or the request approver."
author: "markwahl-msft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessPackageResourceAttributeQuestion resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Resource that defines the question provided to an end user, for the purpose of obtaining an attribute value to be passed to the end system or the request approver.

Inherits from [accessPackageResourceAttributeSource](../resources/accesspackageresourceattributesource.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |Question|accessPackageQuestion|The question asked in order to get the value of the attribute|

Reviewer 2: Fixing the case
```suggestion
|question|accessPackageQuestion|The question asked in order to get the value of the attribute|
```
